README.md

```diff
@@ -1,3 +1,5 @@
+[![Continuous Integration](https://camo.githubusercontent.com/23ee7a697b291798079e258bbc25434c4fac4f8b/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f50726f7465637465645f62792d486f756e642d6138373364312e737667)](https://houndci.com)
+
```

Comment:
Even though HoundCI does not provide a badge, I admire your initiative of using a static badge. Good work.

Reply:
Thanks